### DISCUSSION TITLE
Solved the API Issue and completed the assignment.

### DISCUSSION THREAD

**Discussion by KStar-coder** [hiteshchoudhary/chai-aur-react#84] [Show and tell]
You can visit my repository for reference : https://github.com/KStar-coder/currency-converter

**Comment by abdussamadmalik** [comment]
API isn't working

**Reply by deepaksoam002** [reply]
https://cdn.jsdelivr.net/npm/@fawazahmed0/currency-api@latest/v1/currencies/${currency}.json       use that url
